src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/AnalyzerConfigOptionsExtensions.cs

```diff
@@ -55,5 +52,15 @@ public static bool TryGetEditorConfigOption<T>(this AnalyzerConfigOptions analyz
             value = default!;
             return false;
         }
+
+        public static bool IsAnalysisLevelGreaterThanOrEquals(this AnalyzerConfigOptions analyzerConfigOptions, int minAnalysisLevel)
```

Comment:
This is an existing method that just has been refactored to an internal extension method so it can be invoked from DiagnosticHelper type as well. No functionality changes here.